web-frontend/src/pages/TransferCoins/SendCoins/SendForm.tsx

```diff
@@ -153,6 +160,33 @@ export function SendForm(props: SendFormProps) {
     <Spinner size={12} customClass="inline-block" />
   );
 
+  useEffect(() => {
+    setError({ recipient: '' });
+
+    if (mnsExtension.test(recipient)) {
+      const inputMns = recipient.replace(mnsExtension, '');
+      resolveDns(inputMns);
+    } else {
+      setMnsAddressCorrelelation(false);
+      resetTargetMnsAddress();
+    }
+
+    if (targetMnsAddress && checkAddressFormat(targetMnsAddress)) {
+      setMnsAddressCorrelelation(true);
+    } else {
+      setMnsAddressCorrelelation(false);
+    }
```

Comment:
I guess that could be simplified like: 
```suggestion
    setMnsAddressCorrelelation(targetMnsAddress && checkAddressFormat(targetMnsAddress));
```